libs/utils/src/auth.rs

```diff
@@ -40,6 +40,10 @@ pub enum Scope {
     /// Allows access to storage controller APIs used by the scrubber, to interrogate the state
     /// of a tenant & post scrub results.
     Scrubber,
+
+    /// This scope is used for communication with other storage controller instances.
+    /// At the time of writing, this is only used for the step down request.
+    Peer,
```

Comment:
This Scope enum is used across all components, so let's give a controller-specific scope a more specific name like ControllerPeer

Reply:
Great point. Done in https://github.com/neondatabase/neon/pull/9695/commits/af8a544374841f1608c70195b7c8ec5204d161be